### ISSUE TITLE
Dropdown does not say how many items it has, or announce the value change

### ISSUE THREAD

**Issue by abettadapur** [OfficeDev/office-ui-fabric-react#2654]
### Bug Report

- __Package version(s)__: 4.37.0
- __Browser and OS versions__: Edge/Narrator

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No)

Requested priority: (Blocking, High, Normal, Low) Normal

Products/sites affected: (if applicable)

#### Describe the issue:

When focused on a dropdown input in Edge, Narrator does not read the selected value, instead, it only reads "Editable combo box". If I use the up and down arrow keys to toggle between items, it tries to read the selected item, but is quickly cut off with "Editable combo box."

This issue does not repro in Chrome with Narrator, it reads "[Value] Editable combo box"

**Comment by betrue-final-final** [comment]
This still reproes:

1. Open the component page in Edge
2. Start Narrator
3. Press caps lock + space to turn off scan mode
4. Keyboard to dropdown
5. Arrow down

Result: Value changes, but nothing is said, and the control says it has zero items.
Expect: Control to say how many items it has, and announce the value change.

**Comment by cliffkoh** [comment]
This is covered by #5354.
